ground_truth_slam/plugin.cpp

```diff
@@ -35,7 +35,6 @@ class ground_truth_slam : public plugin {
 		}
 
 		pose_type* true_pose = new pose_type{_m_sensor_data_it->second};
-		true_pose->time = datum->time;
```

Comment:
Do we no longer need the pose's timestamp or did we never use it in the first place?

Reply:
Eh, I added it back.